src/Service/Parser/SingleColumnSingleValueParser.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Service\Parser;
+
+use Lmc\ApiFilter\Constant\Filter;
+
+class SingleColumnSingleValueParser extends AbstractParser
+{
+    public function supports(string $rawColumn, $rawValue): bool
+    {
+        return is_string($rawColumn) && !$this->isTuple($rawColumn) && !is_array($rawValue) && !$this->isTuple($rawValue);
```

Comment:
`is_string(rawColumn)` evaluates always to string. I think you can remove it.

Reply:
good point, it was there, because from the beginning it was `mixed`